TSC-Charter.md

```diff
@@ -176,10 +187,42 @@ discussion will continue.
 For all votes, a simple majority of all TSC members for, or against, the issue
 wins. A TSC member may choose to participate in any vote through abstention.
 
+While the results of all votes must be made public, the actual individual
+ballots cast for most votes may be made public or confidential at the discretion
+of the TSC Chair. However, individual ballots for recertification or removal of
+TSC members must remain confidential. The names of members voting or abstaining
+in all votes must be made public with a clear indication given of whether the
+individual voted or abstained.
+
+Following the completion of all votes, a public statement must be made via
+GitHub specifying the results of the vote. Contextual detail about why the
+vote was held, including a listing of the specific questions voted on must be
+included in the statement.
+
 Note that, in addition to requiring a simple majority vote of the TSC, all
 changes to this charter are also subject to approval from the Node.js
 Foundation board.
 
+### Votes to Remove Members
+
+Should a vote to remove a member result in a tie (50%/50% split of non
+abstaining participants) the matter shall be referred to a Foundation
+appointed independent third party mediator for arbitration.
+
+At any point prior to calling for a vote, the individual member in question, the
+individuals opening the request, or the TSC Chair may request that the issue
+be referred to a Foundation appointed third party mediator for
+arbitration.
```

Comment:
Yeah, I've been stewing on that. The ideal is that it wouldn't have to go to mediation at all. It should also be a goal to avoid having to vote. So perhaps it should be simply if consensus cannot be reached, the matter goes to mediation. The challenge with that is that it encourages members to not take a position, e.g. I don't want to deal with this so abstaining from a clear position forces mediation. I don't know how to solve this yet.